src/Uno.UWP/ApplicationModel/Contacts/ContactPicker.wasm.cs

```diff
@@ -0,0 +1,38 @@
+﻿#nullable enable
+
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.Serialization.Json;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.ApplicationModel.Contacts.Internal;
+using Uno.Foundation;
+
+namespace Windows.ApplicationModel.Contacts
+{
+	public partial class ContactPicker
+	{
+		private const string JsType = "Windows.ApplicationModel.Contacts.ContactPicker";
+
+		private static Task<bool> IsSupportedTaskAsync()
+		{
+			var isSupportedString = WebAssemblyRuntime.InvokeJS($"{JsType}.isSupported()");
+			return Task.FromResult(bool.TryParse(isSupportedString, out var isSupported) && isSupported);
+		}
+
+		private async Task<Contact?> PickContactTaskAsync()
+		{
+			var pickResult = await WebAssemblyRuntime.InvokeAsync($"{JsType}.pickContacts(false)");
```

Comment:
Codacy found an issue: [Remove the unused local variable 'pickResult'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6378321)